### DISCUSSION TITLE
Make babashka available through winget

### DISCUSSION THREAD

**Discussion by huyhoang160593** [babashka/babashka#1709] [Ideas]
I know that with window, you can directly download the binary or using chocolately for install the cli, but with the support of zip file [here](https://github.com/microsoft/winget-cli-restsource/issues/117), and I don't want to install another package manager but the native one on window, can we provide a way to automatic zip the binary and ship though winget ?

**Comment by borkdude** [comment]
You can also install bb via scoop:

https://github.com/littleli/scoop-clojure

**Reply by huyhoang160593** [reply]
@borkdude  Scoop is good and all but there will be another package manager for me to manage, and winget is the official package manager for mircrosoft too, so it should be better if this one available in winget

**Reply by borkdude** [reply]
I personally only maintain the brew package and have off-loaded the maintenance of package managers to others. Is this something you would be interested in maintaining?
